src/java/org/apache/nutch/fetcher/FetcherThread.java

```diff
@@ -695,14 +696,23 @@ private ParseStatus output(Text key, CrawlDatum datum, Content content,
           int validCount = 0;
 
           // Process all outlinks, normalize, filter and deduplicate
+          
+          // NUTCH-2413 Apply filters or normalizers only if configured
+          URLFilters urlFiltersForOutlinks = null;
+          if (conf.getBoolean("parse.filter.urls", true))
```

Comment:
Maybe better read the configuration in the constructor of FetcherThread and save it in a instance variable?